### PR TITLE
Add messages vitals1

### PR DESCRIPTION
**Pull request by lebbiesahr**
Adds in Level of consciousness title

### CODE REVIEW

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -129,6 +129,7 @@ mirebalais.vitals.ratioMuacHC.title=(Calculated) MUAC/HC
 mirebalais.vitals.ratioMuacHC.instructions= If Mid-Upper Arm Circumference (MUAC) and Head circumference (HC) are recorded, the ratio (MUAC/HC) will be calculated.
 mirebalais.vitals.ratioMuacHC.display=Patient's MUAC/HC ratio is:
 mirebalais.vitals.zscore.title=Zscore
+mirebalais.vitals.response.title=Level of consciousness
```
Comment (brandones) [inline api/src/main/resources/messages.properties]: Let's give this a more intuitive name, like `mirebalais.vitals.responsiveness.title` or `mirebalais.vitals.loc.title` or `mirebalais.vitals.levelOfConsciousness.title`

Comment (lebbiesahr) [reply api/src/main/resources/messages.properties]: So response is the term used in the concept dictionary - so kept it for consistency

Comment (brandones) [reply api/src/main/resources/messages.properties]: It's a bad name. Can you link me to where you're talking about? A concept on a server, or the SL concept spec sheet?

Comment (lebbiesahr) [reply api/src/main/resources/messages.properties]: changed to LevelOfConsciouness

File: api/src/main/resources/messages.properties [hunk 2]
```diff
@@ -129,7 +129,7 @@ mirebalais.vitals.ratioMuacHC.title=(Calculated) MUAC/HC
 mirebalais.vitals.ratioMuacHC.instructions= If Mid-Upper Arm Circumference (MUAC) and Head circumference (HC) are recorded, the ratio (MUAC/HC) will be calculated.
 mirebalais.vitals.ratioMuacHC.display=Patient's MUAC/HC ratio is:
 mirebalais.vitals.zscore.title=Zscore
-mirebalais.vitals.response.title=Level of consciousness
+mirebalais.vitals.LevelOfConsciousness.title=Level of consciousness
```
Comment (brandones) [inline api/src/main/resources/messages.properties]: The normative casing here is `camelCase`, i.e., initial letter small. But we'll call this good for now.

**Pull request merged**